ros_buildfarm/templates/devel/devel_create_tasks.Dockerfile.em

```diff
@@ -73,7 +73,8 @@ cmd = \
     ' --os-code-name ' + os_code_name + \
     ' --arch ' + arch + \
     ' --distribution-repository-urls ' + ' '.join(distribution_repository_urls) + \
-    ' --distribution-repository-key-files ' + ' ' .join(['/tmp/keys/%d.key' % i for i in range(len(distribution_repository_keys))])
+    ' --distribution-repository-key-files ' + ' ' .join(['/tmp/keys/%d.key' % i for i in range(len(distribution_repository_keys))]) + \
+    ' --env-vars ' + ' ' .join(env_vars)
```

Comment:
This breaks from the form with the others because the argument dict is copied directly to the data passed to the template in https://github.com/ros-infrastructure/ros_buildfarm/blob/ba90d748bae49252bcd6edb97eed11951017ec3f/scripts/devel/run_devel_job.py#L60.

I'd be tempted to add it to the dict using the same name as the others for better grep-ability. If I did so, would it be preferable to also remove the env_vars key from the dict?